.circleci/config.yml

```diff
@@ -15,14 +15,33 @@ jobs:
             gcloud auth activate-service-account --key-file=${HOME}/gcloud-service-key.json
             gcloud --quiet config set project dev-qlik-core
             gcloud --quiet config set compute/zone europe-west2-a
-            gcloud --quiet container clusters get-credentials core-scaling
       - run:
           name: Accept EULA
           command: grep -rl AcceptEULA ./qlik-core | xargs sed -i 's/AcceptEULA=no/AcceptEULA=yes/g'
+      - run:
+          name: Set Disk Name For Test Cluster
+          command: grep -rl my-disk ./nfs-volumes | xargs sed -i "s/my-disk/$DISK_NAME_TEST/g"
+      - run:
+          name: Deploy test cluster
+          environment:
+            DISK_NAME: "test-circleci-disk"
```

Comment:
Not sure if I can assigne these env. variables from another variables. Example: 
DISK_NAME: $DISK_NAME_TEST
Any idea?